zebROS_ws/src/teleop_joystick_control/src/teleop_joystick_comp_2023.cpp

```diff
@@ -0,0 +1,1454 @@
+// Toggle between rotation using rightStickX and rightTrigger - leftTrigger
```

Comment:
It might be premature to add this now, but if we do, the right thing to do is remove everything related to 2022 mechanisms from the code

Reply:
Done